app/controllers/application_controller.rb

```diff
@@ -30,12 +60,26 @@ def current_user
     super || User.new(guest: true)
   end
 
-  private
+  def handle_error(exception, status, format = params[:format])
+    log_error(exception)
 
-  def render_error_page(status)
+    if format == 'json'
+      render_json_error_response(exception, status)
+    else
+      render_html_error_response(exception, status)
+    end
+  end
+
+  def render_html_error_response(exception, status)
```

Comment:
Unused method argument - `exception`. If it's necessary, use `_` or `_exception` as an argument name to indicate that it won't be used.